packages/storybook/stories/va-input-telephone.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  getWebComponentDocs,
+  propStructure,
+  StoryDocs,
+} from './wc-helpers';
+
+const inputTelephoneDocs = getWebComponentDocs('va-input-telephone');
```

Comment:
I wonder why the `VaSelect` event is showing up in Storybook without a description 🤔 

![Screenshot 2025-05-15 at 8 45 42 AM](https://github.com/user-attachments/assets/1dd7c0d2-765e-44a3-9b65-34d60fc46f68)

Reply:
@jamigibbs - fixed (vaSelect was from combo-box and has been suppressed)